.github/FUNDING.yml

```diff
@@ -0,0 +1,3 @@
+# These are supported funding model platforms
+
+custom: [https://paypal.me/trcsoftware]
```

Comment:
You added 60 NPE-inviting lines of code to a for 20 years existent code base and want to add funding for the project for **your** paypal account? Seriously?